frontend/src/app/libs/admin-role-management/store/users/users-state.ts

```diff
@@ -0,0 +1,7 @@
+import { EntityWrapper } from '../../../auth/models/entity-wraper';
+import { User } from 'src/app/libs/auth/models/user';
+
+export interface UsersState {
+	users: EntityWrapper<User[]>;
+	user: EntityWrapper<User>;
```

Comment:
no need to store current user inside your store, it already stored inside `auth` store